### ISSUE TITLE
[cli] Prevent releasegold:withdraw if a ReleaseGold contract has a cEUR balance

### ISSUE THREAD

**Issue by timmoreton** [celo-org/celo-monorepo#7158]
### Expected Behavior

We currently check to see if a RG contract would self-destruct and take a [cUSD balance with it](https://github.com/celo-org/celo-monorepo/blob/master/packages/cli/src/commands/releasegold/withdraw.ts#L43).  We should add a similar check for cEUR.

### Current Behavior

Someone can send cEUR to a RG contract, then withdraw all remaining CELO, and lose the cEUR.

**Comment by aslawson** [comment]
@gastonponti is this still being worked on or complete?

If not started, I don't think we need to do this now. There's an open question around there being no way to get cEUR or any other token back from the RG contract now. Which means users would be blocked on withdrawing their CELO with this change

**Comment by gastonponti** [comment]
Yes, I've asked that in the cap channel
I didn't create the PR until having a clearer answer.

Are just a few changes, I could add a branch to this issue with something like "to be merged when this was fixed"

**Comment by aslawson** [comment]
Ya on CAP that this was low priority given:

-  its small impact: there really should be no reason someone would try to send cEUR to the contract or the Celo Foundation to deploy a contract with cEUR
- this change would be difficult to apply retroactively to existing contracts, so the value would only be to future RG contracts (almost exclusively cLabs employees now)
-  this is a problem with an ERC-20 token. adding support to withdraw ERC-20 tokens generically could introduce risk around locked Celo. and its not actually possible to check for any ERC-20 balance prior to withdraw.

Given that, I think we can close this as won't do and re-open if it becomes a clear need. We can avoid most possibilities with https://app.zenhub.com/workspaces/devx-backlog-board-6009f3d1a7a39b001018f159/issues/celo-org/celo-monorepo/7157 since most RG users will use the CLI. but the risk with that approach is increasing "transfer" time by adding another check.

slack conversation for reference: https://celo-org.slack.com/archives/CP5V8KF51/p1619561158049100
